main.go

```diff
@@ -85,7 +85,7 @@ func profileToJSON(profile []byte) (string, error) {
 
 func certificateToJSON(p12, key []byte) (string, error) {
 	sKey := strings.TrimSuffix(string(key), "\n")
-	certs, err := pkcs12.DecodeAllCerts(p12, sKey)
+	certs, _, err := pkcs12.DecodeAll(p12, sKey)
```

Comment:
Is this an intended change?

Reply:
Yes. It seems that `DecodeAllCerts` was replaced by `DecodeAll` in the original `go-utils/pkcs12` package at some point. [Here](https://github.com/bitrise-io/go-utils/commit/5dfd5a5f1271cffb2bc0b1cf4c3815139da8175a#diff-160c73da802e19fc807b00614c511a8132ace10b63b6acaefc729342c1155fc7L271) is the exact change.
As far as I understand, this does exactly the same for the certs, and additionally also returns the private keys. However, I would grateful if the Platform team would confirm my assumption.